edit-post/components/sidebar/plugin-post-status-visibility/index.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Defines extensibility lot for the Status & Visibility panel
+ */
+
+import { createSlotFill } from '../../../../components/slot-fill';
```

Comment:
This will include the whole components bundle in the edit-post bundle, to import from other modules, we use `@wordpress/*`. This should be `import { createSlotFille } from '@wordpress/components'`